containers/mit-scheme/.devcontainer/devcontainer.json

```
@@ -0,0 +1,28 @@
+{
+	"name": "MIT-Scheme",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'TARGET_SCHEME_VERSION' to pick an MIT-Scheme version: 11.1
+		"args": { "TARGET_SCHEME_VERSION": "11.1" }
+	},
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": { 
+		"terminal.integrated.shell.linux": "/bin/bash"
+	},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": []
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Uncomment to use the Docker CLI from inside the container. See https://aka.ms/vscode-remote/samples/docker-from-docker.
+	// "mounts": [ "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind" ],
+
+	// Uncomment when using a ptrace-based debugger like C++, Go, and Rust
+	// "runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+	// Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
+	// "remoteUser": "vscode"
```

We want to make sure this runs as non-root so uncommenting this line would be great!

```suggestion
	"remoteUser": "vscode"
```

---

Thanks for your quick reply, done.